.gitignore

```diff
@@ -54,3 +54,8 @@ target
 .project
 .settings
 /ert-profile
+docs/page/lsp-*.md
+/docs/page/settings/
+/docs/page/CHANGELOG.md
+/docs/examples/
+docs/README.md
```

Comment:
We need to take care `ghpages` branch doesn't use that as ignore, but Ithink this will apply for master only

Reply:
You're right that it _should_, if it has the potential to cause headache I can remove it though